make_release/bump-version.nu

```diff
@@ -24,11 +24,20 @@ def main [
 
     ls **/Cargo.toml | each {|file|
         log debug $"bumping ($file.name) from ($version) to ($new_version)"
-        open --raw $file.name | str replace --all --string $'version = "($version)"' $'version = "($new_version)"' | save --force $file.name
+        open --raw $file.name
+            | str replace --all --string $'version = "($version)"' $'version = "($new_version)"'
+            | save --force $file.name
     }
-    ls **/*.nu | each {|file|
-        log debug $"bumping ($file.name) from ($version) to ($new_version)"
-        open --raw $file.name | str replace --all --string $'version = ($version)' $'version = ($new_version)' | save --force $file.name
+
+    let config_files = "crates"
+        | path join "nu-utils" "src" "sample_config" "default_{config,env}.nu"
+        | str expand
```

Comment:
Looks like we found a bug in `str expand`. It doesn't like backslashes, the windows slash.
```nushell
❯ "crates" | path join "nu-utils" "src" "sample_config" "default_{config,env}.nu"
crates\nu-utils\src\sample_config\default_{config,env}.nu
```
```nushell
❯ "crates" | path join "nu-utils" "src" "sample_config" "default_{config,env}.nu" | str expand
╭───┬─────────────────────────────────────────────────╮
│ 0 │ cratesnu-utilssrcsample_configdefault_config.nu │
│ 1 │ cratesnu-utilssrcsample_configdefault_env.nu    │
╰───┴─────────────────────────────────────────────────╯
```
I think this would work in windows though.
```
❯ [crates nu-utils src sample_config 'default_{config,env}.nu'] | str join '/' | str expand
╭───┬─────────────────────────────────────────────────────╮
│ 0 │ crates/nu-utils/src/sample_config/default_config.nu │
│ 1 │ crates/nu-utils/src/sample_config/default_env.nu    │
╰───┴─────────────────────────────────────────────────────╯
``

Reply:
ooh no :sob: 

you might want to file an issue in `nushell/nushell` and ping the author of the `str expand` command :relieved:

Reply:
amazing, thanks :partying_face: